### ISSUE TITLE
is there any plan to move these tutorials to main Avalonia documentation site?

### ISSUE THREAD

**Issue by nsrahmad** [AvaloniaUI/Avalonia.Samples#61]

**Comment by timunie** [comment]
No, there are no such plans. They will stay in this repo. The other way around is planned, moving the tutortials "ToDo-List" and "Music-Store" into this repo with working code to download. 

However, we have links to these samples where useful. If you think there are other places in the docs where we should link a sample, let me know.
